docs/versioned-plugins/inputs/file-v4.4.0.asciidoc

```diff
@@ -82,7 +87,7 @@ was not ideal and a dedicated Read mode is an improvement.
 ==== Compatibility with the Elastic Common Schema (ECS)
 
 This plugin adds metadata about event's source, and can be configured to do so
-in an {ecs-ref}[ECS-compatible] way with <<{version}-plugins-{type}s-{plugin}-ecs_compatibility>>.
+-in an https://www.elastic.co/guide/en/ecs/{ecs_version}/index.html[ECS-compatible] way with <<{version}-plugins-{type}s-{plugin}-ecs_compatibility>>.
```

Comment:
<img width="994" alt="Screen Shot 2022-04-20 at 5 03 53 PM" src="https://user-images.githubusercontent.com/35154725/164323088-cd16b251-c703-4686-bf1d-c29030d042c3.png">

Note that link is resolving to 8.2 branch rather than `current`.